firestore-send-email/integration-test/firestore.indexes.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Looks like these were exported from the extensions-testing project by firebase init. They can probably be replaced with default rules files for the purposes of this test.